docker/project.go

```diff
@@ -15,7 +15,7 @@ import (
 const ComposeVersion = "1.5.0"
 
 // NewProject creates a Project with the specified context.
-func NewProject(context *Context) (project.APIProject, error) {
+func NewProject(context *Context, parseOptions *config.ParseOptions) (*project.Project, error) {
```

Comment:
Good catch! Didn't mean for that to be in this PR. I'll fix and we can move discussion over to #260.